client/state/data-layer/wpcom/sites/plan-transfer/index.js

```diff
@@ -0,0 +1,70 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { dispatchRequestEx } from 'state/data-layer/wpcom-http/utils';
+import { errorNotice, successNotice } from 'state/notices/actions';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { SITE_PLAN_OWNERSHIP_TRANSFER } from 'state/action-types';
+
+const noticeOptions = siteId => ( {
+	duration: 8000,
+	id: `sites-plan-transfer-notice-${ siteId }`,
```

Comment:
That's the right property, I'm always confused because internally it's `noticeId` 🤦‍♂️ 

https://github.com/Automattic/wp-calypso/blob/1764fe922ad54661b27d735941a50a41e12a8c24/client/state/notices/actions.js#L23-L27

Reply:
#25580 🤷‍♂️